src/main/java/lotto/view/UserInput.java

```diff
@@ -0,0 +1,21 @@
+package lotto.view;
+
+import java.util.Scanner;
+
+public class UserInput {
+
+    private static final Scanner sc = new Scanner(System.in);
+    private UserInput() {
+    }
```

Comment:
상수와 생성자의 구분 및 가독성을 위해서 공백 라인을 추가해보면 좋을 것 같습니다.
```suggestion
    private static final Scanner sc = new Scanner(System.in);

    private UserInput() {
    }
```